Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/view/MoreInformationView.kt

```diff
@@ -68,11 +68,7 @@ class MoreInformationView @JvmOverloads constructor(
         val title = binding.titleElement.text
         val subtitle = binding.subtitleElement.text
         val isToggleVisible = binding.toggle.isVisible
```

Comment:
[Code smell] `isToggleVisible` is not used anymore , should be removed as well from the function